src/index.md

```diff
@@ -1,7 +1,6 @@
 ---
 layout: index
 title: Welcome to Workbox
-excerpt: Workbox is a collection of JavaScript libraries for Progressive Web Apps
```

Comment:
It looks like we've lost the `except` that was added in https://github.com/GoogleChrome/workbox-microsite/pull/66